README.md

```diff
@@ -10,14 +10,19 @@ Just the HM Docs is a GitHub Pages template developed for the purpose of quickly
 
 Getting started with Just the HM Docs is simple.
 
-1. Create a [new repository based on Just the HM Docs](https://github.com/humanmade/just-the-hm-docs/generate).
-1. Update `_config.yml` and `README.md` with your project information. [Be sure to update the url and baseurl](https://mademistakes.com/mastering-jekyll/site-url-baseurl/).
-1. Configure a [publishing source for GitHub Pages](https://help.github.com/en/articles/configuring-a-publishing-source-for-github-pages). Your project docs website is now live!
-1. Edit and create `.md` [Markdown files](https://guides.github.com/features/mastering-markdown/) to add more content pages.
+1. Download the [starter theme branch](https://github.com/humanmade/just-the-hm-docs/tree/starter-theme) to get the following files:
+  - the [`_config.yml`](https://github.com/humanmade/just-the-hm-docs/blob/starter-theme/_config.yml) file and the [`Gemfile`](https://github.com/humanmade/just-the-hm-docs/blob/starter-theme/Gemfile)
```

Comment:
Did installing using `remote_theme` not work? I don't recall needing this much configuration for when we set up the webpack helpers docs

Reply:
The theme is set in the `_config.yml` file, and yeah, I think these two things are the only config needed to set up the site. Although it might be good to link to a sample of a working GitHub Actions workflow config also.

Reply:
I think `remote_theme` is only for GitHub hosted themes, which we could do, but since we published the gem, we can just use `theme` to use the published gem. As far as the config options, it depends on what you want to use. I trimmed back a bunch of the options from the [config file](https://github.com/humanmade/just-the-hm-docs/blob/starter-theme/_config.yml) in the `starter theme` branch, but if you want to use the styles for the hm branded callouts, for example, you need those config options. (At least from my testing, happy to find a better way.)